JASP-Engine/JASP/R/descriptives.R

```diff
@@ -195,6 +195,24 @@ Descriptives <- function(jaspResults, dataset, options) {
     }
     .descriptivesScatterPlots(jaspResults[["scatterPlots"]], dataset.factors, variables, splitName, options)
   }
+
+  # Interval plots
+  if (options$descriptivesIntervalPlot) {
+    if(is.null(jaspResults[["IntervalPlots"]])) {
+      jaspResults[["IntervalPlots"]] <- createJaspContainer(gettext("Intervalplots"))
+      jaspResults[["IntervalPlots"]]$dependOn(c("descriptivesIntervalPlot", "splitby"))
+      jaspResults[["IntervalPlots"]]$position <- 11
+    }
+    
+    intervalPlots <- jaspResults[["intervalPlots"]]
```

Comment:
With the change below I do get a plot in the output!
```suggestion
    intervalPlots <- jaspResults[["IntervalPlots"]]
```

Reply:
Perfect!! Me too. How do you like the output?